### PR TITLE
Alert banner

### PR DESCRIPTION
**Pull request by Yukti-work**
Alert banner initial commit

### CODE REVIEW

File: apps/storybook/src/unstable/datacom-alert-banner.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { StoryFn } from '@storybook/react';
+import { DatacomAlertBanner } from '@datacom/endeavour-react';
+
+type AlertBannerProps = React.ComponentProps<typeof DatacomAlertBanner>;
+
+export default {
+  title: 'Alert-Banner',
```
Comment (AngeCast) [inline apps/storybook/src/unstable/datacom-alert-banner.stories.tsx:8]: Should replace the dash in the title here with a space. This title will be visible on storybook (and I'm positive the designers will as you change this when they review this component.)

File: packages/stencil/src/components/datacom-alert-banner/datacom-alert-banner.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  closeIcon = getSvg('close', { class: 'dc-close-icon' });
+
+  render() {
+    return (
+      <Host>
+        <div class="dc-alert-banner">
+          {this.copy}
+          <a href={this.url}>{this.link}</a>
+          {this.closeIcon}
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-alert-banner/datacom-alert-banner.tsx:22]: Shouldn't this close icon actually close the banner on click? You should probably look to add functionality to do that. Maybe wrap this close icon with a button element and then add a click handler on to that.

**Review by AngeCast — CHANGES_REQUESTED** [review]
Hey there. Couple of comments from me. Have a look when you can.

File: packages/stencil/src/components/datacom-alert-banner/datacom-alert-banner.tsx (hunk 2 above)
Comment (sidra2022) [reply packages/stencil/src/components/datacom-alert-banner/datacom-alert-banner.tsx]: Done

File: apps/storybook/src/unstable/datacom-alert-banner.stories.tsx (hunk 1 above)
Comment (sidra2022) [reply apps/storybook/src/unstable/datacom-alert-banner.stories.tsx]: Fixed

**Pull request closed without merging**